packages/kbn-plugin-generator/sao_template/template/translations/ja-JP.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
This will fail when running kibana since we expect a `formats` and `messages` fields in the json.

```suggestion
{
  "formats": {
    "number": {
      "currency": {
        "style": "currency"
      },
      "percent": {
        "style": "percent"
      }
    },
    "date": {
      "short": {
        "month": "numeric",
        "day": "numeric",
        "year": "2-digit"
      },
      "medium": {
        "month": "short",
        "day": "numeric",
        "year": "numeric"
      },
      "long": {
        "month": "long",
        "day": "numeric",
        "year": "numeric"
      },
      "full": {
        "weekday": "long",
        "month": "long",
        "day": "numeric",
        "year": "numeric"
      }
    },
    "time": {
      "short": {
        "hour": "numeric",
        "minute": "numeric"
      },
      "medium": {
        "hour": "numeric",
        "minute": "numeric",
        "second": "numeric"
      },
      "long": {
        "hour": "numeric",
        "minute": "numeric",
        "second": "numeric",
        "timeZoneName": "short"
      },
      "full": {
        "hour": "numeric",
        "minute": "numeric",
        "second": "numeric",
        "timeZoneName": "short"
      }
    },
    "relative": {
      "years": {
        "units": "year"
      },
      "months": {
        "units": "month"
      },
      "days": {
        "units": "day"
      },
      "hours": {
        "units": "hour"
      },
      "minutes": {
        "units": "minute"
      },
      "seconds": {
        "units": "second"
      }
    }
  },
  "messages": {}
}
```